### ISSUE TITLE
No corpus case has ever been read by a person; 13 are in debt

### ISSUE THREAD

**Issue by mstarks01** [mstarks01/work-agent#226]
`evals/BLESSING.md` step 6 is one reading session over a case's `source.md`, `model.json` and every framework's reference set together. **It has never been run, for any of the 13 cases.** All of them were written by an agent.

`tests/test_case_review.py` makes that countable: a case carrying no `review` block in `case.json` is named in `UNREVIEWED`, and a **new** case arriving without one fails. That list is debt, not an exemption — no entry says the omission is acceptable, and it is meant to reach zero.

## Why a lint cannot replace it

Review sitting 01 (#208) found a reference claim asserting the model emits *training data* in a case with no training pipeline — the one failure the eval suite treats as gating, sitting in the standard the gate reads.

A mechanical version of that check fires on **231 of 243 claims**, because a claim is *supposed* to describe an attack in words the system description never uses ("floods", "replays", "impersonates"). Narrowing to the asset vocabulary fails too: training data falls under `business-critical-data`, which that case already carries.

So this class of defect is not mechanically detectable, and a person reading is the only instrument for it.

## What is waiting

**`evals/corpus/01-payments-checkout/REVIEW-02.md`** is written and ready — the first run, and the template for the rest. About an hour.

It asks the question underneath sitting 01: *do these reference sets describe what could actually go wrong with this system?* Recall is measured against them, so a whole class of attack missing means the tool scores full marks for missing it too, and nothing in the repo would say so.

Case 01 first because it is the control — every far-domain recall figure is a delta against it, so an error there moves every comparison the corpus exists to make.

Structure follows sitting 01's lesson about anchoring: Part 1 is the source and a readable model summary and asks for **the reader's own threat list first**; Part 2 is the 21 STRIDE threats; Part 3 is the 17 ASVS records with the narrower question they need. Then one section for what was on the reader's list and not on either. The document ends with the exact `review` block to paste and the line to delete from `UNREVIEWED`.

## The three outcomes, all written into the document

- **Few doubts, nothing important missing** — the control holds, and the suite's recall numbers have a real standard behind them for the first time.
- **A whole class of attack missing** — the serious one. Every recall figure quoted so far needs re-deriving.
- **Several doubts** — the list overstates, inflating the denominator. Cheaper direction, still wrong.

## The remaining 12

One sitting each, same template, generated the same way. Not urgent in the way case 01 is: the control is what every other case is measured against.

## Parity answer

The check is already N-ary — `required_reading` derives from `case.meta.frameworks`, so a review that read one package's reference set leaves the case in debt (#210). A case that gains a third framework's set re-opens its own review with no edit.

**Comment by mstarks01** [comment]
Unchanged by the claim-identity work (#237 through #241) — recording that explicitly so it does not look covered.

The corpus now carries an action verb on all 243 reference claims and the identity rule scores 185/200 against the recorded labels. **None of that is a substitute for this.** Both numbers are agreement with what an agent wrote down, and step 6 is the only instrument that would catch the class of defect review sitting 01 found in case 04 — a reference claim asserting a fact its own model does not hold. A mechanical check for that fires on 231 of 243 claims, which is why `verify_corpus.py` cannot stand in for it.

One thing that did change: `tests/test_verb_coverage.py` now guards a second field, so a step 6 sitting has more to read than it did — the verb on each claim is an assignment somebody should agree with, and it is agent-authored like everything else.

#242 lists this as item 5 for context; **this issue is the one to track it on.**

**Comment by mstarks01** [comment]
PR #247 (merged, with the parity fix in #248) generates the reading document for every case in debt: `evals/corpus/<case>/REVIEW.md`, one per unreviewed case, from the case's own files — sources verbatim, the model as tables, the element notes, your own list first, a mark line per record, and the exact `review` block to paste. `REVIEW-02.md` stays as case 01's hand-written document. Each sitting is now pure reading time; the documents regenerate via `python evals/build_review_docs.py` when a reference set changes.

**Comment by mstarks01** [comment]
**Before anybody spends the hour: the browser path does not clear a case.**
[#634](https://github.com/mstarks01/work-agent/issues/634) has the detail.

`webapp/sitting.py` — the launcher `evals/BLESSING.md` step 6 documents — writes a JSON
submission under `evals/review/submissions/`. `tests/test_case_review.py:136` builds its
verdict from `case.json` → `reviews` alone and never opens that directory. So the rail greys
the row and `--list` drops the case, and the debt this issue counts does not move.

The by-hand path in `evals/BLESSING.md:425` still writes the `reviews` entry, so it does
clear a case today. Either read #634 first, or take the by-hand path until it is ruled.

**Comment by mstarks01** [comment]
**Corrected: the browser path now clears a case.**
[#634](https://github.com/mstarks01/work-agent/issues/634) is fixed and merged
([#635](https://github.com/mstarks01/work-agent/pull/635)).

My earlier note said to take the by-hand path instead. That advice is out of date, and the
by-hand path it pointed at no longer exists. A sitting is now one JSON file under
`evals/review/submissions/`, and `tests/test_case_review.py` reads it through
`evals.review_submission.current_reviews` — the same function the app's rail asks. Clearing a
case and seeing it cleared are the same answer.

So the hour this issue asks for is worth spending, through the documented launcher:

```sh
uv run python webapp/sitting.py
```

The **Contribute** button opens a pull request carrying that one file. It needs no roster line
and writes nothing else. `evals/BLESSING.md` step 6 has the current method, and ADR 24 records
what the change gave up.

**Comment by mstarks01** [comment]
## New evidence that the debt now blocks two lines of work, not one

Two measurements on 2026-09-15, both from runs made for other reasons.

### Extraction: 95% of the model's "errors" are names

Five sweeps of all thirteen cases, 65 extractions ([#925](https://github.com/mstarks01/work-agent/issues/925#issuecomment-5687228732)). Per sweep the model produces 121.8 extra Elements, of which **115.6 could be a blessed Element of the same type under another name** and only **6.2 are named nowhere in their own Source**. Strict `recall` reads 0.525; `sourced_recall`, which credits a name a reader ruled supported, reads 0.785.

`zone_recall` is 0.485 while `zone_partition_agreement` is 0.890 — the model groups the right Elements into the same trust zone and names the zone differently.

#882 already found **eleven corpus Element names that never appear in the Source they were extracted from**, and the scorer charged the model for using the Source's own words. That was eleven names found by audit. Nobody has read the thirteen cases to find out how many more there are.

So: an unknown share of a 0.525 recall figure is the reference's error rather than the model's, and **no instrument in the repo can tell the two apart**. Every extraction number the repo quotes rests on that.

### The assertion layer's acceptance criterion is blocked on the same reading

#926 merged Phases 1 to 3. Its only offline instrument is the projection against the blessed model's control attributes, and **nine of the sixteen predicates in registry version 2 project into no graph field at all** — `credential-sharing`, `authorization-grant` and `mfa-requirement` among them, which are the facts that issue exists to capture.

Measuring those needs a reference catalog. An agent-authored one is not ground truth, so Phase 6's "required-fact recall across the full reviewed denominator" cannot be computed until a person sits with a case.

### Why this is the cheapest thing available

It costs reading time and no money. Everything else on the extraction list costs a paid run that produces a number nobody can interpret until this is done:

- A prompt edit (#295) moves naming, which moves `comparison_coverage`, which moves the denominator of every agreement figure.
- A scorer change needs to know whether the reference or the model is wrong.
- #926 Phase 6 cannot start.

`evals/BLESSING.md` step 6 is the procedure. `tests/test_case_review.py` counts the debt.

**Comment by mstarks01** [comment]
Closed by #1120. Every case in `evals/corpus/` is read.

One Case Sitting on 2026-09-21 cleared all 13: **130 own-list threats written
blind**, then **all 344 recorded claims marked** — 253 STRIDE, 91 ASVS.

| mark | count |
| --- | ---: |
| `agree` | 233 |
| `unsure` | 61 |
| `reject` | 26 |
| `duplicate` | 24 |

Nothing was left unmarked, so every case records. `unreviewed_cases` returns
nothing and `webapp/sitting.py --list` prints an empty list.

**26 `missing` entries**, each traced to the own-list item it came from. They
are what the reader wrote before the sets opened and could not find a row for
— replay and rollback (cases 03, 11, 12), a forged internal handoff (06),
per-operation role separation (09), stored comment injection (10), credentialed
CORS and WebSocket hijack (13), firmware rollback and public-download abuse
(02). A sitting cannot edit a reference set, so these travel as prose and are
a maintainer's to act on.

**What a mark is not.** It answers a claim as written. It does not make the set
exhaustive — that is what the `missing` list is for, and the 26 entries say
where this reader thought it is not.

#1121 then audited the prose the sitting made false: `evals/BLESSING.md` opened
on "no case has a merged Case Sitting", `evals/README.md` called the reference
sets "unreviewed" twice, and `UNREVIEWED` held 13 spent entries. All three are
corrected, and the count now prints from the reader rather than sitting in
prose.
